src/cbor.h

```diff
@@ -144,6 +148,7 @@ struct CborEncoder
         ptrdiff_t bytes_needed;
     };
     const uint8_t *end;
+    size_t added;
```

Comment:
Yes, BC break is fine, we've only got a static library anyway. SC is more important.